### DISCUSSION TITLE
problem with the other page with github.io domain

### DISCUSSION THREAD

**Discussion by Kashumy** [community/community#57750] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

# can i have a repository named e.g. cojystal.github.io in my profile

and this repository will be available at cojystal.github.io
even though I have one repo named kashumy.github.io
is it possible ?

I will be grateful for your response ☺️

**Comment by airtower-luna** [comment, marked as answer]
No. You get one user site (`<username>.github.io` domain) that matches your username. You can crate a `cojystal.github.io` repository and enable GitHub Pages, but then it'll be published at `https://kashumy.github.io/cojystal.github.io/` (in a subdirectory matching the repository name, like any other repository).

Alternatives:
* Create an organization with the name you want, and create an organization site.
* Use a custom domain.

**Reply by Kashumy** [reply]
thanks for you'r answer, i've been struggling with this for a long time
☺️

**Reply by Kashumy** [reply]
I need to make a different user so that e.g. cojystal will be available under the domain cojystal.github.io :/

**Reply by airtower-luna** [reply]
Careful with that, the GitHub ToS allow only one free user per person. That's why I mentioned the organization alternative above. :sweat_smile: 

Another alternative would be to use a custom domain (outside `github.io`), but then you'd need to buy that domain first (from your registrar of choice).

**Reply by Kashumy** [reply]
thank you for Attention me and helping me.  do you know how to make organization with web site ?

where can i find out about it
